docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -34,11 +34,12 @@ All actions executed on a host are tracked in the host’s response actions hist
 [[isolate-a-host]]
 == Isolate a host
 
-.Isolate a host from a case alert
+.Isolate a host from a detection alert
```

Comment:
Just curious, is "detection alert" the official way to refer to any alert?  The key detail for response actions, such as host isolation, to be possible from the alert is that it originated from an Endpoint.  Do we want to reflect that in any way in the text?

Reply:
Yes, "detection alert" means any alert in Elastic Security. We often shorten to "alerts" but use the longer form for first mentions, headings, etc.

For response actions in general, we have a [Requirements bullet](https://security-docs_2763.docs-preview.app.elstc.co/guide/en/security/master/response-actions.html) saying that Elastic Defend is required, but we don't have anything as specific on the [host isolation](https://security-docs_2763.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-ov.html) page. It's kind of implied, but I think it'd help to be more explicit and try to prevent any possible confusion. I'll add a bullet to the host isolation requirements...